src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/AnalyzeContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public class AnalyzeContext
+    {
+        public AnalyzeContext(IUpgradeContext context)
```

Comment:
I'm assuming this is a WIP type :)

Reply:
Yes this is WIP, needs to be flushed out what more I can add to AnalyzeContext